.kitchen.vagrant.yml

```diff
@@ -1,44 +0,0 @@
----
-driver:
```

Comment:
@chris-rock I switched from erb approach to the kitchen local approach. Can you please have a look?